### PR TITLE
Add account management features

### PR DESCRIPTION
**Pull request by DinithiDiaz**
## Purpose
This PR is to add account management features in the CDS Toolkit.

This adds the documentation of the following features:

- Business Nominated Representative
- Secondary User Instructions
- Secondary User Ceasing
- Disclosure Options Management

This will fix 

- https://github.com/wso2/docs-open-banking/issues/735
- https://github.com/wso2/docs-open-banking/issues/736
- https://github.com/wso2/docs-open-banking/issues/741
- https://github.com/wso2/docs-open-banking/issues/742

### CODE REVIEW

File: en/docs/learn/configuring-data-sharing-for-secondary-users-configuration.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+## Enabling Secondary User Accounts
+
+Follow the below instructions to enable the secondary user account feature:		
+
+1. Open the `<IS_HOME>/repository/conf/deployment.toml` file.
+
+2. Add the following configurations:
+
+    ```toml
```
Comment (AmilaSamith) [inline en/docs/learn/configuring-data-sharing-for-secondary-users-configuration.md:11]: remove "toml"

File: en/docs/learn/ceasing-data-sharing-for-secondary-users-configuration.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Configurations
+
+## Enabling the ceasing data sharing feature for secondary users
+
```
Comment (lalaji) [inline en/docs/learn/ceasing-data-sharing-for-secondary-users-configuration.md:4]: Can you change PR title and the description to be more meaningful. Mention what are the exact features you have covered in this PR of documenting.

File: en/docs/learn/configuring-data-sharing-for-secondary-users.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+!!! info
+    This is only available as a WSO2 Update from **WSO2 Open Banking Identity Server CDS Toolkit Level 1.0.0.8** onwards. For more information on updating, see [Getting WSO2 Updates](../install-and-setup/setting-up-servers.md#getting-wso2-updates).
+
+## Integrating with the bank back-end
+
+### Secondary Account Update API
+
+The Secondary Account Update API is defined below.
+
+The bank needs to invoke this endpoint to update the open banking solution regarding status changes happening on secondary user accounts. This endpoint persists the received secondary user-related information in the `OB_ACCOUNT_METADATA` table. When persisting the `accountId` of the secondary account and `secondaryUserId` parameters are used as a combined key and metadata will be stored as key-value pairs.
```
Comment (AmilaSamith) [inline en/docs/learn/configuring-data-sharing-for-secondary-users.md]: lets correct to "user related"

Comment (AmilaSamith) [reply en/docs/learn/configuring-data-sharing-for-secondary-users.md]: Lets capitalize as "accountID" and "secondaryUserID"

File: en/docs/learn/configuring-data-sharing-for-secondary-users.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+   "updatedTimestamp": 1683569588,
+   "validityPeriod": 1683629555,
+   "consentAttributes": {
+      "ExpirationDateTime": "1683629555",
+      "commonAuthId": "1a25530b-cf2a-4341-8b27-5fd0a2d5557c",
+      "sharing_duration_value": "60000"
+      "consentId": "85d14688-abe5-4cc0-98c3-7f6af1d0a5e2",
+      "consentMappingResources": [....],
+      "secondaryAccountInfo": [{
+         "secondaryAccountUser": ”user1 @gold.com @carbon.super”,
```
Comment (AmilaSamith) [inline en/docs/learn/configuring-data-sharing-for-secondary-users.md]: should corrected to ”user1@gold.com @carbon.super”

File: en/docs/learn/configuring-data-sharing-for-secondary-users.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+A sample request is given below:
+
+```
+curl --location --request PUT 'https://<IS_HOME>:9446/api/openbanking/account-type-mgt/account-type-management/secondary-accounts' \
+--header 'Authorization: Basic YWRtaW5Ad3NvMi5jb206d3NvMjEyMw==' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+   "data": [
+      {
+         "accountId":"30080098763500",
```
Comment (AmilaSamith) [inline en/docs/learn/configuring-data-sharing-for-secondary-users.md]: change accountID" and "secondaryUserID" in request body as well.

File: en/docs/learn/ceasing-data-sharing-for-secondary-users-configuration.md (hunk 2 above)
Comment (DinithiDiaz) [reply en/docs/learn/ceasing-data-sharing-for-secondary-users-configuration.md:4]: This is fixed.

File: en/docs/learn/configuring-data-sharing-for-secondary-users.md (hunk 5 above)
Comment (DinithiDiaz) [reply en/docs/learn/configuring-data-sharing-for-secondary-users.md]: Fixed via https://github.com/wso2/docs-ob-toolkit-cds/pull/78/commits/cae48427b88d4c488acaa5414de54c8e21565690.

File: en/docs/learn/configuring-data-sharing-for-secondary-users.md (hunk 4 above)
Comment (DinithiDiaz) [reply en/docs/learn/configuring-data-sharing-for-secondary-users.md]: Fixed via https://github.com/wso2/docs-ob-toolkit-cds/pull/78/commits/cae48427b88d4c488acaa5414de54c8e21565690.

File: en/docs/learn/configuring-data-sharing-for-secondary-users-configuration.md (hunk 1 above)
Comment (DinithiDiaz) [reply en/docs/learn/configuring-data-sharing-for-secondary-users-configuration.md:11]: This is required for the docs formatting.

File: en/docs/learn/configuring-data-sharing-for-secondary-users.md (hunk 3 above)
Comment (DinithiDiaz) [reply en/docs/learn/configuring-data-sharing-for-secondary-users.md]: Fixed via https://github.com/wso2/docs-ob-toolkit-cds/pull/78/commits/cae48427b88d4c488acaa5414de54c8e21565690.

File: en/docs/learn/configuring-data-sharing-for-business-nominated-representatives-configuration.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
+# Enabling the Business User Profile
+
+There are 2 profile types as `individual` and `organization`. Following are the 2 ways to get the profile type and you can configure.
```
Comment (imesh94) [inline en/docs/learn/configuring-data-sharing-for-business-nominated-representatives-configuration.md:3]: We have to mention that there can be multiple organization profiles according to the consumer's accounts

File: en/docs/learn/configuring-data-sharing-for-business-nominated-representatives-configuration.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+
+- Enabling the profile selection page
+
+- Profile selection based on the bank's backend response
+
+### Enabling the profile selection page
+
+After a bank customer initiates a consent flow with an Accredited Data Recipient (ADR), they are redirected to the Data Holder’s authentication process. Usually, upon successful authentication, the Data Holder redirects the customer to the account selection step and provides the relevant list of bank accounts the bank customer has the authority to share data.
+
+By default, the profile selection page is disabled. If the profile selection feature is enabled, the bank customer can select the profile as either `business` or `individual` during the consent authorization flow.
```
Comment (imesh94) [inline en/docs/learn/configuring-data-sharing-for-business-nominated-representatives-configuration.md:13]: This is enabled by default

**Review by lalaji — APPROVED** [review]

File: en/docs/learn/configuring-data-sharing-for-business-nominated-representatives-configuration.md (hunk 6 above)
Comment (lalaji) [reply en/docs/learn/configuring-data-sharing-for-business-nominated-representatives-configuration.md:3]: let's fix this in next PR. @imesh94

File: en/docs/learn/configuring-data-sharing-for-business-nominated-representatives-configuration.md (hunk 7 above)
Comment (lalaji) [reply en/docs/learn/configuring-data-sharing-for-business-nominated-representatives-configuration.md:13]: let's fix this in next PR. @imesh94

**Pull request merged**